.gitignore

```diff
@@ -1,4 +1,3 @@
-dist
```

Comment:
This should still be included in the ignore I believe, was there a specific reason to remove it?